packages/components/search-field/src/search-field.scss

```diff
@@ -0,0 +1,15 @@
+:host {
+  --_button-padding: 2px;
+
+  padding-inline-end: calc(var(--sl-space-input-text-field-inline-md) - var(--_button-padding));
+}
+
+button {
+  align-items: center;
+  appearance: none;
+  background: transparent;
+  border: 0;
+  cursor: pointer;
+  display: inline-flex;
+  padding: var(--_button-padding);
+}
```

Comment:
This button is able to get the focus when using keyboard navigation, but it doesn't use the default focus ring (styling), causing it to look like this in Max for example:
<img width="75" alt="image" src="https://github.com/sl-design-system/components/assets/1351956/0489cf3b-dc2c-42c3-9d1e-c1b0472ac2da">

Reply:
There is of course also a case to be made for it not being focusable with the keyboard. (although I'm not sure if the 'esc' functionality is commonly known)

Reply:
I've changed it to use a small `<sl-button>`. And the button is keyboard focusable.